packages/tools/pie-css/test/css/css.spec.ts

```diff
@@ -0,0 +1,50 @@
+import * as fs from 'fs';
+import * as util from 'util';
+import * as path from 'path';
+import cssValidator from 'w3c-css-validator';
+import {
+    describe,
+    it,
+    expect,
+} from 'vitest';
+
+const readFile = util.promisify(fs.readFile);
+
+async function getCssContent (filePath: string): Promise<string> {
+    try {
+        const data = await readFile(filePath, 'utf8');
```

Comment:
Nit: We don't need a variable here as it's not being used anywhere.

IMO it's already clear what's being returned given the function name / return type.

Reply:
hope you don't mind but I'd prefer to keep this for now :)